monarchmoney/monarchmoney.py

```diff
@@ -489,6 +489,239 @@ async def get_account_holdings(self, account_id: int) -> Dict[str, Any]:
             variables=variables,
         )
 
+    async def get_account_history(self, account_id: int) -> Dict[str, Any]:
+        """
+        Gets historical account snapshot data for the requested account
+
+        Args:
+          account_id: Monarch account ID as an integer
+
+        Returns:
+          json object with all historical snapshots of requested account's balances
+        """
+
+        query = gql(
+            """
+            query AccountDetails_getAccount($id: UUID!, $filters: TransactionFilterInput) {
+              account(id: $id) {
+                id
+                ...AccountFields
+                ...EditAccountFormFields
+                isLiability
+                credential {
+                  id
+                  hasSyncInProgress
+                  canBeForceRefreshed
+                  disconnectedFromDataProviderAt
+                  dataProvider
+                  institution {
+                    id
+                    plaidInstitutionId
+                    url
+                    ...InstitutionStatusFields
+                    __typename
+                  }
+                  __typename
+                }
+                institution {
+                  id
+                  plaidInstitutionId
+                  url
+                  ...InstitutionStatusFields
+                  __typename
+                }
```

Comment:
Is there value in removing some the fields and fragments from this query?  Obviously no issue in using the same as the Monarch interface, just a thought for reducing data weight given the intent of the function.